### DISCUSSION TITLE
unocss.css does not inject properly on nuxt3

### DISCUSSION THREAD

**Discussion by AQian0** [unocss/unocss#2868] [Q&A] [answered]
I'm trying to use the unocss vue-scoped mode in nuxt3

![image](https://github.com/unocss/unocss/assets/61798525/1629be60-f63c-4713-b5a0-1772742b7fc7)
![image](https://github.com/unocss/unocss/assets/61798525/4e6fc69d-7630-4cb3-bca7-c8fcfab495bb)

But there is an error like this

![image](https://github.com/unocss/unocss/assets/61798525/bb7febb5-b4b3-4e4a-993a-dc4f94bb8b08)

I noticed that there is no unocss.mjs in the generated .nuxt folder.However, according to the documentation, unocss.css is automatically injected in nuxt3.

![image](https://github.com/unocss/unocss/assets/61798525/452e0adc-9d84-4e13-924e-7d9c4f260ea3)

I'm assuming there may be something wrong with unocss, is there any way to fix it?

My English is not very good, translation provided by DeepL.

**Comment by zyyv** [comment]
We temporarily close this due to the lack of enough information.
Please provide a [minimal reproduction](https://stackoverflow.com/help/minimal-reproducible-example) to reopen the issue.
Thanks.

> [Why reproduction is required](https://antfu.me/posts/why-reproductions-are-required)

**Reply by AQian0** [reply]
Minimal reproduction have been added.

**Comment by AQian0** [comment]
Here's a demo of my code, I reproduced the problem on CodeSandbox.

https://codesandbox.io/p/sandbox/intelligent-tess-v2zyqv

**Comment by zyyv** [comment, marked as answer]
Fixed on #2873
